contribs/sharing/src/main/java/org/matsim/contrib/sharing/service/SharingService.java

```diff
@@ -8,17 +8,23 @@
 import org.matsim.core.mobsim.framework.MobsimAgent;
 
 public interface SharingService {
-	public Id<SharingService> getId();
+	Id<SharingService> getId();
 
-	public IdMap<SharingVehicle, SharingVehicle> getVehicles();
+	IdMap<SharingVehicle, SharingVehicle> getVehicles();
 
-	public IdMap<SharingStation, SharingStation> getStations();
+	IdMap<SharingStation, SharingStation> getStations();
 
-	public void pickupVehicle(SharingVehicle vehicle, MobsimAgent agent);
+	void pickupVehicle(SharingVehicle vehicle, MobsimAgent agent);
 
-	public void dropoffVehicle(SharingVehicle vehicle, MobsimAgent agent);
+	void reserveVehicle(MobsimAgent agent,SharingVehicle vehicle);
```

Comment:
For ease of use, maybe adjust the order of the arguments to the other methods (first vehicle, then agent)